index.js

```diff
@@ -93,12 +121,10 @@ app.post('/quote', (req, res) => {
           },
           {
             "type": "context",
-            "elements": [
-              {
-                "type": "mrkdwn",
-                "text": "Quote by _Quote author_ from *Movie name*"
-              }
-            ]
+            "elements": [{
```

Comment:
In situations like these where there's a lot of repeated code, it may be good for us to generate this array with a function. We may in the future want to show 4 options instead of 3, and it would be nice to be able to make that change easily. OR, for example, we may add a feature in the future where a user gets to _choose_ how many suggestions they get back, so it would be good to have this already be dynamic just in case.

This doesn't need to be fixed for _this_ PR, but maybe for the next one.